--- a/setup.py
+++ b/setup.py
@@ -17,10 +17,10 @@
       url='TODO',
       license = 'GPL-3.0+',
       packages=['thermod'],
-      scripts=['bin/thermod'],
+      #scripts=['bin/thermod'],
       install_requires=['python-daemon >= 2.0.5', 'jsonschema >= 2.3.0'],
-      test_suite='nose.collector',
-      tests_require=['nose >= 1.3.4', 'requests >= 2.4.3'],
+      #test_suite='nose.collector',
+      #tests_require=['nose >= 1.3.4', 'requests >= 2.4.3'],
 )
 
-# vim: fileencoding=utf-8
\ No newline at end of file
+# vim: fileencoding=utf-8
