--- a/web/index.php
+++ b/web/index.php
@@ -19,7 +19,7 @@
  */
 
 	// hostname (or IP address) and port where Thermod socket is listening on
-	// see section 'socket' in file etc/thermod.conf
+	// see section 'socket' in file /etc/thermod/thermod.conf
 	$HOST = 'localhost';
 	$PORT = '4344';
 ?>
--- a/etc/thermod.conf
+++ b/etc/thermod.conf
@@ -12,11 +12,11 @@
 
 [scripts]
 # The following options can be full path to scripts (that return a JSON string
-# like examples script in `samples` folder) or the value 'PiPinsRelay' if you
-# want to use one (or more) relay connected to Raspberry Pi GPIO pins. If you
-# choose to use Raspberry the value 'PiPinsRelay' must be set in all three
-# options and the section PiPinsRelay below must be adjusted to meet your
-# Raspberry Pi setup.
+# like examples script in /usr/share/doc/thermod/samples folder) or the value
+# 'PiPinsRelay' if you want to use one (or more) relay connected to Raspberry Pi
+# GPIO pins. If you choose to use Raspberry the value 'PiPinsRelay' must be set
+# in all three options and the section PiPinsRelay below must be adjusted to meet
+# your Raspberry Pi setup.
 switchon = /etc/thermod/switch-heating --on -j -s -q
 switchoff = /etc/thermod/switch-heating --off -j -s -q
 status = /etc/thermod/switch-heating --status -j -s -q
