--- a/etc/thermod.conf
+++ b/etc/thermod.conf
@@ -7,12 +7,13 @@
 [global]
 enabled = false
 debug = false
-timetable = /etc/thermod/timetable.json
+timetable = /var/lib/thermod/timetable.json
 interval = 30
 
 [scripts]
 # These variables must be full path to executable scripts that return a JSON
-# string after having completed their job. See samples folder for some examples.
+# string after having completed their job. See /usr/share/doc/thermod/samples
+# folder for some sample scripts.
 thermometer = /etc/thermod/get-temperature
 switchon = /etc/thermod/switch-heating --on -j -s --noinfo
 switchoff = /etc/thermod/switch-heating --off -j -s --noinfo
--- a/web/index.php
+++ b/web/index.php
@@ -19,7 +19,7 @@
  */
 
 	// hostname (or IP address) and port where Thermod socket is listening on
-	// see section 'socket' in file etc/thermod.conf
+	// see section 'socket' in file /etc/thermod/thermod.conf
 	$HOST = 'localhost';
 	$PORT = '4344';
 ?>
